lib/search/global/profiles.js

```diff
@@ -67,6 +67,23 @@ module.exports = client => async (term = '', query = {}) => {
           boost: 5
         }
       }
+    },
+    {
+      match: {
+        email: {
+          query: term,
+          fuzziness: 'AUTO'
+        }
+      }
+    },
+    {
+      wildcard: {
+        email: {
+          value: `${term}*`,
+          boost: 5,
+          rewrite: 'scoring_boolean'
```

Comment:
This rewrite method seems to give the best results from some manual testing...

The docs https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-term-rewrite.html have a warning that says
> This method can cause the final bool query to exceed the clause limit in the indices.query.bool.max_clause_count setting. If the query exceeds this limit, Elasticsearch returns an error.

So not sure if this will cause issues.